packages/data-connector/src/firebase/converters/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { FirebaseMap } from '../models/utils';
+
+export const firebaseMapToPairArray = <S>(
```

Comment:
En firebase [los arrays no estan recomendados](https://firebase.googleblog.com/2014/04/best-practices-arrays-in-firebase.html) así que con esta pequeña utilidad sirve tener un mapa de objetos (pedidos por ejemplo) cuyo índice sea su id en la base de datos, y conertirlo a un array, en este caso es un arrays de pares [id, pedido], para luego poder mapear ese array a un arrays de pedidos